modules/tensor_mechanics/include/utils/RankTwoTensor.h

```diff
@@ -102,6 +102,61 @@ class RankTwoTensor
   /// Gets the value for the index specified.  Takes index = 0,1,2, used for const
   Real operator()(unsigned int i, unsigned int j) const;
 
+  /// Get the xx value
+  Real & xx();
+
+  /// Get the xx value, const
+  Real xx() const;
+
+  /// Get the xy value
+  Real & xy();
+
+  /// Get the xy value, const
+  Real xy() const;
+
+  /// Get the xz value
+  Real & xz();
+
+  /// Get the xz value, const
+  Real xz() const;
+
+  /// Get the yx value
+  Real & yx();
+
+  /// Get the yx value, const
+  Real yx() const;
+
+  /// Get the yy value
+  Real & yy();
+
+  /// Get the yy value, const
+  Real yy() const;
+
+  /// Get the yz value
+  Real & yz();
+
+  /// Get the yz value, const
+  Real yz() const;
+
+  /// Get the zx value
+  Real & zx();
+
+  /// Get the zx value, const
+  Real zx() const;
+
+  /// Get the zy value
+  Real & zy();
+
+  /// Get the zy value, const
+  Real zy() const;
+
+  /// Get the zz value
+  Real & zz();
+
+  /// Get the zz value, const
+  Real zz() const;
+
+
```

Comment:
This adds a lot of code for just a little bit of syntactic sugar. I'm not a big fan of this. It breaks the consistency we have with the rank four tensor (and if we were to add these methods for that one we'd need even more of these functions).